README.md

```diff
@@ -2,10 +2,12 @@
 Ground Support Equipment Software for the Cornell Rocketry Team including fill station and ground server
 
 ## Ground Server
-To start the ground server, run
-```sudo snap install docker```
-```docker pull zachgarcia42/ground-server-image```
-```docker run -it -d -p 80:80 zachgarcia42/ground-server-image```
+
+First, install Docker for your specific computer. Then, to start the ground server run:
+```shell
+docker pull ghcr.io/cornellrocketryteam/ground-software
+docker run -it -d -p 80:80 zachgarcia42/ground-server-image
```

Comment:
Should update the run line to be ghcr.io/cornellrocketryteam/ground-software. Or ghcr.io/cornellrocketryteam/ground-server if we change the image name.